### PR TITLE
Added support for statslite

### PR DESCRIPTION
**Pull request by liach**
[Statslite](https://github.com/Minecrell/statslite) is a plugin developed by [Minecrell](https://github.com/Minecrell). It provides a client for mcstats, and it is already implemented in [FoxGuard](https://git.io/vglVQ).
P.S. I am very interested in this project and I want to be a collaborator.

### CODE REVIEW

**Comment by Flibio** [comment]
What are the benefits to adding support for this?

**Comment by liach** [comment]
I think it is up. I want to call @minecrell to explain for statslite.

**Comment by Flibio** [comment]
I've used PluginMetrics in a few plugins before, but discontinued it due to the unreliable website. I like plugin statistics tracking, but don't like PluginMetrics, for the reason stated above.

**Comment by Minecrell** [comment]
While the service is indeed down from time to time, I'm sure Hidendra does his best to keep in up most of the time. You need to keep in mind there is an incredible load on the server, if about 150k servers contact Plugin Metrics most with multiple plugins every 15 minutes. There are probably like 5k or more requests served every second and considering the service is completely free we should be grateful Hidendra has been maintaining this for such a long time even though he seemed to be quite busy with other projects within the last year.

While I understand that the downtime is sometimes a bit annoying in the last time there it was often just down for like 5 minutes or something, possibly if too many requests are coming in. I doubt you'll find any other free alternative which is more reliable, and still provides the same set of features as MCStats does.

I'm not affiliated with the project in any way, however in my experience as a plugin developer who has been using it for the last 3 years it has always been a reliable and extremely useful tool most of the time. statslite is specifically written in a way it does not visibly log connection errors because they happen from time to time, however at the end you'll still have accurate statistics most of a time with a system people have known for a long time and habe learned to trust (it is fully open source, with something only on your server for example we couldn't tell what you're actually doing with the data). statslite profides a shared configuration so server owners could (theoretically) easily opt-out of the statistics for the plugins using statslite.

File: build.gradle [hunk 1]
```diff
@@ -18,23 +18,31 @@ repositories {
...
   main {
     java.srcDir 'src'
     compileClasspath += api.output
   }
 }
 
+dependencies {
+  compile "org.spongepowered:spongeapi:${project.apiVersion}"
+  apiCompile "org.spongepowered:spongeapi:${project.apiVersion}"
+  shade 'net.minecrell:statslite-sponge:0.1'
```
Comment (Minecrell) [inline build.gradle]: You should update this for the latest statslite release (0.2), I broke some things for consistency, should be better now

**Comment by Flibio** [comment]
@Minecrell You provide a very convincing argument for PluginMetrics. Thanks for taking the time to write that.

@liach As soon as you perform the changes recommended, I'll merge the PR.

**Comment by liach** [comment]
statslite updated.

**Comment by Minecrell** [comment]
You need to update the imports, the package changed.

**Comment by liach** [comment]
Done. Also updated the maven artifact name.

**Pull request merged**